### PR TITLE
adding file structure

### PR DESCRIPTION
**Pull request by Noorin99**
Relates #3

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "axios": "^0.23.0",
```
Comment (ahmadfarid-059) [inline client/package.json:9]: you should install tailwind library

File: client/src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (ahmadfarid-059) [inline client/src/App.css:1]: remove unused css code

File: client/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^5.3.0",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.0.1"
```
Comment (ahmadfarid-059) [inline client/package.json:15]: you should remove webvitals we won't use it

File: client/src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (ahmadfarid-059) [inline client/src/App.jsx:1]: I suggest using .js  ending

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+package-lock.json
```
Comment (ahmadfarid-059) [inline .gitignore:1]: we should add  .env to .gitignore file

**Review by ahmadfarid-059 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (Noorin99) [reply .gitignore:1]: Changes are done :)

**Review by ahmadfarid-059 — APPROVED** [review]

**Pull request merged**